package.json

```diff
@@ -4,10 +4,21 @@
   "description": "Comprehensive tooling for working with OpenAPI definitions",
   "license": "MIT",
   "author": "ReadMe <support@readme.io> (https://readme.com)",
+  "sideEffects": false,
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "require": "./dist/index.js",
+      "import": "./dist/index.mjs"
+    },
+    "./package.json": "./package.json"
```

Comment:
A lot of packages are you this export for example preact or nextjs

This can be used to support more then one version of a package


import pkg from "oas/package.json"

pkg.version....